workspaces/adventure-pack/goodies/typescript/Math.lcm/index.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Math.lcm", () => {
+  it.each([
+    [4, 5, 20],
+    [6, 8, 24],
+    [21, 6, 42],
+    [13, 17, 221],
+    [0, 5, 0],
+    [5, 0, 0],
+    [0, 0, 0],
+    [-5, 15, 15],
+    [7, -3, 21],
+    [-6, -8, 24],
+    [1000000007n, 1000000009n, 1000000016000000063n],
+    [2n, 1000000007n, 2000000014n],
+    [9999991n, 99999959n, 999998690000369n],
+    [123456, 789012, 8117355456],
+    [15, 25, 75],
+  ])("Math.lcm(%p, %p) === %p", (a, b, expected) => {
+    expect(Math.lcm(Number(a), Number(b))).toBe(Number(expected));
```

Comment:
Can we avoid the Number casts, and express the numbers above not in `bigint` form? If the numbers are too big to express as regular Numbers, the result is likely not meaningful anyway.

Reply:
ok, I will remove the offending test case (line 17 actually gave me a warning like this: 
```
 This number literal will lose precision at runtime  no-loss-of-precision
```